rollupsync/oracle.go

```
@@ -1,19 +1,24 @@
 import (
+	"bytes"
 	"context"
 	"errors"
 	"time"
 
 	authzv1beta1 "cosmossdk.io/api/cosmos/authz/v1beta1"
+	ibcprotoclient "github.com/cometbft/cometbft/proto/ibc/core/client/v1"
+	ibcprotopmlcs "github.com/cometbft/cometbft/proto/ibc/lightclients/tendermint/v1"
+	cmtproto "github.com/cometbft/cometbft/proto/tendermint/types"
 	"github.com/cometbft/cometbft/rollupsync/provider"
 	"github.com/cometbft/cometbft/types"
```

golangci-lint (1.62.2)

ST1019: package "github.com/cometbft/cometbft/types" is being imported more than once

(stylecheck)